Resources/Prototypes/Catalog/Fills/Lockers/suit_storage.yml

```diff
@@ -212,6 +215,8 @@
   - type: StorageFill
     contents:
         - id: OxygenTankFilled
+        - id: NitrogenTankFilled # DeltaV - slimefriendly
```

Comment:
This was removed on Wizden "to make slime more unique", this needs to be put into every locker if this goes into Sec Lockers; in a seperate PR

This also needs a changelog entry